lib/components/PublicAuthPrompt.vue

```diff
@@ -0,0 +1,224 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog :buttons="dialogButtons"
+		class="public-auth-prompt"
+		data-cy-public-auth-prompt-dialog
+		is-form
+		:can-close="false"
```

Comment:
deprecated and removed in v9

```suggestion
		no-close
```